vision_model/model_base.py

```diff
@@ -0,0 +1,32 @@
+from typing import Dict, List, Union
+import torch
+from .plugin_base import PluginBase
+
+
+class ModelBase(torch.nn.Module):
```

Comment:
Do we want a base model interface for inference as well?

Reply:
For concrete inference model interface wrt different tasks like IC, OD, Caption, image-text matching, where do you see them fit? We would like to share it in vision-benchmark and potentially with researchers as well

Reply:
This interface supports both training and inference. To make this interface flexible for new tasks, task-specific formats are not included in the code. We expect the Predictor module to generate prediction results in the expected format.

Reply:
We still need standardization like https://dev.azure.com/pinjin/Vision-Benchmark/_git/Vision-Benchmark?path=/README.md&version=GBmaster&line=186&lineEnd=187&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents to ask collaborators/researchers to conform to speed up benchmark exp, just wondering where do you think they belong

Reply:
They are described in the last section of the README. The predictor module should return the results in the defined formats. We are not defining base predictor for each class. Most of the cases, there will be just tensor shape difference, which cannot be written as type hints, hence not so useful.